namui-cli/electron/src/main.js

```diff
@@ -33,6 +33,7 @@ function createWindow() {
         height: 720,
         webPreferences: {
             preload: path.join(__dirname, "preload.js"),
+            webSecurity: false,
```

Comment:
please set `allowRunningInsecureContent` false to prevent other security problem.
![image](https://user-images.githubusercontent.com/3580430/182217252-f3ab818e-fbd1-49cd-910e-614f55f70970.png)

> https://www.electronjs.org/docs/latest/api/browser-window#new-browserwindowoptions

Reply:
Ah! got it

Reply:
[b0d124f](https://github.com/NamseEnt/namseent/pull/296/commits/b0d124f4e989c3f8a1f7c3833f3b64e152e80dc8) fixed